trillian_admin_api.proto

```diff
@@ -21,6 +21,7 @@ option java_outer_classname = "TrillianApiProto";
 package trillian;
 
 import "trillian.proto";
+import "github.com/google/trillian/crypto/keyspb/keyspb.proto";
```

Comment:
As best as I can tell, you can't import protos from a different directory with a relative path like that. It causes the compilation error `cannot find package "crypto/keyspb"`.